client/components/LoginPage.tsx

```diff
@@ -1,44 +1,93 @@
 import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { 
+  TextField, 
+  Box, 
+  Typography, 
+  InputAdornment,
+  IconButton,
+  Button
+} from "@mui/material";
+import {
+  Visibility,
+  VisibilityOff
+} from "@mui/icons-material";
+
 import { useLoginMutation } from "../features/api";
 
 const LoginPage = () => {
   const [login, { isLoading, isError, data }] = useLoginMutation();
 
+  const navigate = useNavigate();
+
+
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
+  const [showPassword, setShowPassword] = useState(false);
+  const [showConfirmPassword, setShowConfirmPassword] = useState(false);
+
+  const handleClickShowPassword = () => setShowPassword((show) => !show);
+  const handleMouseDownPassword = (event: React.MouseEvent<HTMLButtonElement>) => {
+    event.preventDefault();
+  };
 
   const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
     e.preventDefault();
     login({ email, password });
+    navigate("/");
   };
 
   return (
-      <form onSubmit={handleSubmit}>
-        <h1>Login</h1>
-        {isError && <p>Oops, there was an error logging in. Try again?</p>}
-
-        <label htmlFor="email">Email</label>
-        <input
-          type="email"
-          id="email"
-          name="email"
-          onChange={(e) => setEmail(e.target.value)}
-          value={email}
-        />
-
-        <label htmlFor="password">Password</label>
-        <input
-          type="password"
-          id="password"
-          name="password"
-          onChange={(e) => setPassword(e.target.value)}
-          value={password}
-        />
-
-        <button disabled={isLoading} type="submit">
-          Login
-        </button>
-      </form>
+    <Box 
+      component="form"
+      onSubmit={handleSubmit}
+      autoComplete="off"
+    >
+      <Typography component="h1">Login</Typography>
+      {isError && <Typography 
+        paragraph={true}
+      >
+        {/*TODO: Change this to an alert*/}Oops, there was an error creating an account. Try again?
+      </Typography>}
+
+      <TextField
+        id="email"
+        label="Email"
+        type="email"
+        required
+        onChange={(e) => setEmail(e.target.value)}
+        value={email}
+      />
+
+      <TextField
+        id="password"
+        label="Password"
+        type={showPassword ? "text" : "password"}
+        required
+        InputProps={{
+          endAdornment:
+            <InputAdornment position="end">
+              <IconButton
+                aria-label="toggle password visibility"
+                onClick={handleClickShowPassword}
+                onMouseDown={handleMouseDownPassword}
+                edge="end"
+              >
+                {showPassword ? <VisibilityOff /> : <Visibility/>}
+              </IconButton>
+            </InputAdornment>
+        }}
+        onChange={(e) => setPassword(e.target.value)}
```

Comment:
same nit: no need for extra parens
```suggestion
        onChange={e => setPassword(e.target.value)}
```